app/assets/javascripts/alert.js

```diff
@@ -0,0 +1,28 @@
+(function () {
+  var float_alert;
+  float_alert = typeof exports !== 'undefined' && exports !== null ? exports : this;
```

Comment:
@mssola I'll try to figure out what I did wrong. I did a simple test in the browser console and thought it would have no impact.